exe/Server.hs

```diff
@@ -195,15 +199,22 @@ opts = mkOpts
        <> showDefault
        <> value 8000
         )
+    <*> switch
+        ( long "version"
+       <> help "show the version"
+        )
   where
-    mkOpts h pgp u pw db p = Opts (ConnectInfo h pgp u pw db) p
+    mkOpts h pgp u pw db = Opts (ConnectInfo h pgp u pw db)
 
 
 -- * Main
 
 main :: IO ()
 main = do
     opts' <- execParser allOpts
+    when (optsShowVersion opts') $ do
+        putStrLn $ "radicle-server version " <> showVersion version
```

Comment:
Is this useful? We haven't changed cabal-file versions at all so far, and if we're generating this for each new commit on master, we definitely are going to have lots of *different* binaries with the same version even if we do start bumping sometimes.

Reply:
I added this to have something to run to check that `radicle-server` is correctly linked when executed in the container (see https://github.com/oscoin/radicle/pull/259/files#diff-889dc80f39676827dc606c84c4baf1e0R13). However I realize that we can just use `radicle-server --help >/dev/null`. I’ll make the change.